### ISSUE TITLE
DBMS_OUTPUT not outputting in debug mode

### ISSUE THREAD

**Issue by markgargan** [flyway/flyway#1929]
Hi Axel,

Just in case, what would be really useful if it isn't way too much hassle would to be able to provide a DBMS_OUTPUT filename as a flyway property where the output would redirect. That way we wouldn't need to filter the statements out of the DEBUG. 

###### What version of Flyway are you using?
<flyway.version>5.0.7</flyway.version>
<oracle.driver.version>12.1.0.1</oracle.driver.version>
        	

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin, SBT plugin, ANT tasks)
Maven plugin

###### What database are you using (type & version)?
Oracle 12c

###### What operating system are you using?
Windows 7 Enterprise

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, and the command you ran)
Ran the following SQL through the plugin 
DECLARE
  somesql VARCHAR2(10);
BEGIN

  somesql := 'Hello';
  DBMS_OUTPUT.put_line('SQL :-' || somesql);

END;
/

###### What did you expect to see?
the output 'SQL :-hello' output on a line in it's own right like 
DEBUG: SQL :-hello

###### What did you see instead?
'Found statement' and 'Executing SQL' debug statements both contain the actual DBMS_OUTPUT statement itself but not the resolved output itself. 

If I pre-pend
SET SERVEROUTPUT ON
as the first line in the file both with and without a ';' at the end of the line it throws an exception

Exception :
Line       : 1
Statement  : SET SERVEROUTPUT ON

        at org.flywaydb.maven.AbstractFlywayMojo.execute(AbstractFlywayMojo.java:603)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
        ... 19 more
Caused by: java.sql.SQLSyntaxErrorException: ORA-00922: missing or invalid option

        at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)

in relation to the SET SERVEROUTPUT ON command regardless of whether I put a semi-colon at the end of the command or not. 

If I remove the line the maven process completes, the file is migrated but hello only appears in the

**Comment by axelfontaine** [comment]
Could it be that you are on the Community Edition? (SQL*Plus statement support, like `SET SERVEROUTPUT ON` is a Flyway Pro feature)

**Comment by markgargan** [comment]
ah! Sorry Axel, my apologies if I am, I didn't realise there was two different editions. I presume we're on the community edition. I've pulled the 5.0.7 artifacts down from the mvnrepository.com?

**Comment by axelfontaine** [comment]
No worries Mark. This is new since 5.0.0. For Pro, the best way is to start with downloading the trial at https://flywaydb.org/download/ . The download contains a script to install the artifacts in your local repo. Should you choose to decide to buy the Pro license, just be aware that customers also get access to a private Maven repo with full source code access.

Please confirm it is working with the trial for you.

**Comment by markgargan** [comment]
hi Axel, thanks very much for your prompt responses to this issue, it's much appreciated! I can confirm that the output is appearing as expected once I used the pro trial license. 

[INFO] DB: SQL Hello
[DEBUG] Successfully completed migration of schema...

We've adopted flyway as our version control across the board for our db version control so I'll let my manager know that the PRO license is available. 

Thanks,
Mark.
